lib/api/unittest/CDataFrameAnalyzerFeatureImportanceTest.cc

```diff
@@ -33,6 +34,8 @@ using TDoubleVec = std::vector<double>;
 using TStrVec = std::vector<std::string>;
 using TMeanVarAccumulator = ml::maths::CBasicStatistics::SSampleMeanVar<double>::TAccumulator;
 
+const std::string SHAP_PREFIX{"feature_importance."};
```

Comment:
Is this used?  It looks like you've used the identical constant from `maths::CDataFrameRegressionModel` instead.

Reply:
Good catch. Indeed, it's not used. Fixed now.